applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.css

```diff
@@ -16,6 +16,38 @@ thead input {
     display: none;
 }
 
+/*New CSS Starts here*/
+
+ul.pagination {
+    justify-content: center !important;
+}
+
+
+div.dataTables_filter > label > input {
+    width: 518px !important;
+    height: 37px;
+    top: -1365px;
```

Comment:
these values are suspect - i think there should be a better way to align this then the massive negative values